FastLM-Backend/app.py

```
@@ -0,0 +1,1469 @@
+        # 연결된 사용자 워크스페이스 관계 모두 삭제
+        UserWorkspace.query.filter_by(workspace_id=workspace_id).delete()
+        
+        # 연결된 공지사항들의 스케줄드 작업들 삭제
+        notices = Notice.query.filter_by(workspace_id=workspace_id).all()
+        for notice in notices:
+            # 스케줄러에서 작업 삭제
+            scheduled_jobs = ScheduledJob.query.filter_by(notice_id=notice.id).all()
+            for job in scheduled_jobs:
+                try:
+                    scheduler.remove_job(job.job_id)
+                except:
```

Flake8 (7.2.0)

[error] do not use bare 'except'

(E722)